crates/namada/src/vm/wasm/memory.rs

```diff
@@ -171,10 +171,10 @@ pub fn write_vp_inputs(
 /// it will try to grow the memory.
 fn check_bounds(memory: &Memory, base_offset: u64, len: usize) -> Result<()> {
     tracing::debug!(
-        "check_bounds pages {}, data_size {}, base_offset + len {}",
+        "check_bounds pages {}, data_size {}, base_offset {base_offset}, len \
+         {len}",
```

Comment:
probably not the most appropriate PR to make this change, but I think renaming `base_offset` to `base_address`, and `len` to `offset` would make more sense. we're calculating an offset over some base address, which we will dereference at some later point in time

Reply:
I'm ok with it :) - done in [b8ca7af](https://github.com/anoma/namada/pull/2401/commits/b8ca7af34d223166be53f161be2ec940cda9a48b)